nebula/ui/components/VPNMobileStatusBarModifier.qml

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.5
+
+import Mozilla.VPN 1.0
+
+QtObject {
+    id: statusBarModifier
+
+    enum StatusBarTextColor {
+        Light,
+        Dark
+    }
```

Comment:
If the enum is in the QML file, C++ needs to do what you have implemented: a quint8. And basically, we end up doing a mapping (int -> enum value). I wonder if it's cleaner to add an enum in C++ in `src/theme.h` and use the QT macro `Q_ENUM`. In this way, here we can do:

`property var statusBarTextColor: VPNTheme.StatusBarDarkTextColor` (or any better name).

We then propagate the VPNTheme.<NameOfTheEnum>.